src/common/transformations/src/transformations/control_flow/unroll_if.cpp

```diff
@@ -42,6 +42,9 @@ bool ov::pass::UnrollIf::run_on_model(const std::shared_ptr<ov::Model>& f) {
         auto body = (cond_value[0]) ? if_node->get_then_body() : if_node->get_else_body();
         auto input_descriptions = if_node->get_input_descriptions(static_cast<int>(!cond_value[0]));
         auto output_descriptions = if_node->get_output_descriptions(static_cast<int>(!cond_value[0]));
+        // copy rt info before reconnection
+        for (auto& op : body->get_ops())
+            copy_runtime_info({op, if_node}, op);
```

Comment:
why do we need to copy rt info from "op" to "op"?
I think it should be
copy_runtime_info(if_node, op);

Reply:
> why do we need to copy rt info from "op" to "op"? I think it should be copy_runtime_info(if_node, op);

`copy_runtime_info(if_node, op)` replaces rt_info of `op` by rt_info from `if_node`

Reply:
> I don't think so, `copy_runtime_info` calls this function inside ![image](https://user-images.githubusercontent.com/21181928/270603194-b8d96c91-640e-45ec-91f3-f181dee96266.png) As I can see, only repeating attributes will be overwritten, other attributes should remain

I have added test checks which are failed with `copy_runtime_info(if_node, op)`

Reply:
I've double checked. 
`copy_runtime_info({op, if_node}, op);`  
calls `merge` method of `FusedNames` attribute for `{op, if_node}` and doesn't call it otherwise.
Looks correct. Resolved.